spacy/pipeline/entity_linker.py

```diff
@@ -284,11 +311,9 @@ def initialize(
         nO = self.kb.entity_vector_length
         doc_sample = []
         vector_sample = []
-        for eg in islice(get_examples(), 10):
+        examples = self._augment_examples(islice(get_examples(), 10))
```

Comment:
Why exactly 10 here? I don't remember the specifics about this anymore.

Reply:
We do this everywhere just to have some data for `model.initialize()` so that Thinc can infer the correct `nI`/`nO` dimensions.